packages/sirius-web/backend/sirius-web-application/src/main/java/org/eclipse/sirius/web/application/project/data/versioning/services/DefaultProjectDataVersiongRestService.java

```diff
@@ -70,11 +94,64 @@ public List<RestCommit> getCommits(IEditingContext editingContext) {
     }
 
     /**
-     * The default implementation does not allow several commits per project.
+     * The default implementation does not allow several commits per project. An amend commit is then executed.
      */
     @Override
-    public RestCommit createCommit(IEditingContext editingContext, Optional<UUID> branchId) {
-        return null;
+    public RestCommit createCommit(IEditingContext editingContext, Optional<UUID> branchId, List<RestDataVersion> changes) {
```

Comment:
Please let me know if this implementation should be moved into a dedicated class or anything else that could improve the readability.